### ISSUE TITLE
I need some help

### ISSUE THREAD

**Issue by HsLOL** [lehduong/torch-warmup-lr#1]
@lehduong Thanks for your work!
when I use the command **pip install git+git://github.com/lehduong/torch-warmup-lr.git**, I find there is no torch_warmup_lr in my conda list, but it's in pip list. And I can't import torch_warmup_lr in my py file. How can i solve this problem?

Thanks

**Comment by lehduong** [comment]
Hi, have you tried cloning the repo and run `python setup.py`?

**Comment by HsLOL** [comment]
> Hi, have you tried cloning the repo and run `python setup.py`?

Thanks for your reply!
I have ran the **python setup.py  install** and I got the **torch_warmup_lr-1.0.0-py3.6.egg** file, but I don't know how to use it to install by conda, and I didn't get some useful information by searching some websites. 

Could you please tell me how to do after running python setup.py 

Thanks a lot!
